index.html

```diff
@@ -0,0 +1,243 @@
+      </style>
+      <script
+      src="https://kit.fontawesome.com/675259b560.js" crossorigin="anonymous">
+    </script>
+    <title>Document</title>
+</head>
+<body>
+    <header>
+            <section id="home">
+                <nav id="navbar" class="open">
+                    <div class="container">
+                        <a href='#' class="logo">My Logo</a>
+                        <div class="boton"><i class="fa-solid fa-bars fa-2xs"></i></div>
+                    </div>
+                </nav>
+            </section>
+    </header>
+    <main>
+        <section class="portfolio">
+            <div>
+                <h1>Hey There. I'm Jonathan.</h1>
+            </div>
+            <div>  
+                <h2>I'm a Software Developer</h2>
+            </div>
+            <div>
+                <p>I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                        dont hesitate to contact me.</p>
+            </div>
+            
+            <div class="icons">
+                <a href="">
+                    <img src="Assets/Github.svg" alt="Github">
+                </a>
+                <a href="">
+                    <img src="Assets/Linkedin.svg" alt="Linkedin">
+                </a>
+                <a href="">
+                    <img src="Assets/Icon - Angelist.svg" alt="Angelist">
+                </a>
+                <a href="">
+                    <img src="Assets/Twitter.svg" alt="Twitter">
+                </a>
+                <a href="">
+                    <img src="Assets/Medium.svg" alt="Medium">
+                </a>
+            </div>
+        </section>
+        <section id="experience">
+            
+            <h2 class="title">My Recent Works</h2>        
+            <img class="vector" src="Assets/Vector.svg" alt="">        
+            <div>
+                <img  class="placeholder" src="Assets/Img Placeholder.svg" alt="">
+                
+                <div>
+                    <h2 class="work">Multi-Post Stories</h2>
+                </div>
+                <div>
+                    <p class="paragraph">A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.</p>
+                </div>
+                <div class="teclast">
+                    <div class="css-button">CSS</div>
+                    <div class="html-button">HTML</div>
+                    <div class="boot-button">Bootstrap</div>
+                    <div class="ruby-button">Ruby</div>
+                </div>
+                <div class="orange-button">
+                    <div class="see-project">See project</div>
+                </div>
```

Comment:
- [ ] Please 🙏🏻  follow the best practices by using correct semantics in your *HTML* code. As I can see that you have used the `div` tag instead of the **button** tag which is not good at all because `div` is not a semantics tag only used for `decoration` purposes whereas on the other hand `button` is a semantics tag and have the meaning, for example, **See project** should be a button so that later on when any person clicks on it then he/she will be able to see your *PopUp Model Window* 💻 to get more detail about your project.

- [ ] Please adhere to the Figma design because your design has some issues related to spacing which makes your design looks unprofessional or not look good to the eye. Kindly have a look at the below table to understand the problem in a better way through visualization.

###  Design Comparison

<table>
<tr>
<th>Your Design</th>
<th>Figma Design</th>
</tr>

<tr>
<td>

![Screenshot from 2023-06-06 13-07-00](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/b4b1428f-73a6-47a1-8ef7-8dc8572bbd9b)


</td>

<td>

![Screenshot from 2023-06-06 13-06-46](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/6563ee52-9ffb-4051-a9a4-4ccc04d62a1a)


</td>
</tr>
</table>


#### Requirement

![image](https://github.com/sinruidotec/Portfolio-part-2-Microverse/assets/66476329/641d0c3f-f77f-4683-a5d1-b0c8780bdbe6)